app/api/v1/models.py

```diff
@@ -0,0 +1,133 @@
+import base64
+import random
+
+db = [
+    {
+        "user": [],
+        "parcel": [],
+        "tokens": []
+    }
+    ]
+parcel = []
+
+
+class Model:
+    """This is the model class to manipulate data"""
+    def __init__(self):
+        """initialzation for our data"""
+        self.db = list(db)
```

Comment:
Doing `list(db)` is not necessary as `db` is already a list. Initialize it to `db` instead.